doc/source/whatsnew/v0.20.0.rst

```diff
@@ -374,7 +374,7 @@ For example, after running the following, ``styled.xlsx`` renders as below:
    df.iloc[0, 2] = np.nan
    df
    styled = (df.style
-             .applymap(lambda val: 'color: %s' % 'red' if val < 0 else 'black')
+             .applymap(lambda val: 'color:red;' if val < 0 else 'color:black;')
```

Comment:
does this change the practical output?

Reply:
The previous version was supposed to yield in ['color:red', 'color:black'] but erroneously yielded in ['color:red', 'black']. This will have silently ignored 'black' as bad-css and not done anything - probably the text was still rendered black by default!

The new code has a `raise` when it detects no valid attribute-value pair, so this was causing an `IPython directive error` in tests.

Reply:
ok, pls add a whatsnew note about this (you can speak generally, e.g. now raise an error if invalid CSS is passed)